bin/lib/zopen-update-cacert

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+#
+# Updates the cacert in zopen
+#
+export utildir="$( cd "$(dirname "$0")" >/dev/null 2>&1 && pwd -P )"
+
+. "${utildir}/common.inc"
+
+ZOPEN_CA="${utildir}/../../cacert.pem"
+
+printHelp() 
+{
+  args=$*
+  echo "zopen-update-cacert will update your $ZOPEN_CA file to the latest curl CA certificates extracted from Mozilla"
+}
+
+getCaCertFromCurl()
+{
+  if ! curl -L -o "/tmp/$LOGNAME.cacert.pem" "https://curl.se/ca/cacert.pem"; then
+    printError "curl command could not download $url"
```

Comment:
where does LOGNAME get set? Since you are writing to /tmp, should you be adding $$ or something into the name to ensure that you don't collide with another user?

Reply:
LOGNAME gets set by the shell.  I'll add a random val in between